5.1/start.sh

```diff
@@ -80,6 +80,14 @@ export NOMINATIM_QUERY_TIMEOUT=10
 export NOMINATIM_REQUEST_TIMEOUT=60
 echo "Warming finished"
 
+# Wait for PostgreSQL to be ready
+echo "Waiting for PostgreSQL to be ready..."
+until pg_isready -h localhost -p 5432 -U postgres; do
```

Comment:
what if db is external? this will break. also at the very least id put this behind flag

Reply:
> also at the very least id put this behind flag

You are right. Maybe someone relies on faulty requests when starting/restarting the container. Will push an update later today.